src/tests/issues.targets

```diff
@@ -186,6 +186,9 @@
         <ExcludeList Include="$(XunitTestBinBase)/JIT/jit64/regress/vsw/373472/**">
             <Issue>Allocates large contiguous array that is not consistently available on 32-bit platforms</Issue>
         </ExcludeList>
+        <ExcludeList Include="$(XunitTestBinBase)/JIT/Regression/JitBlue/Runtime_34170/Runtime_34170/*">
```

Comment:
hmm.. this might disable it for non-crossgen2 cases as well. Might make sense to discuss with @trylek if there is a way to disable for crossgen2+arm32 only?

Reply:
At this point we don't have a pre-existing section in issues.targets specifically targeting Crossgen2+arm32 but it's trivial to add that e.g. after the general Crossgen2 section by just copying the conditional <code>ItemGroup</code> and modifying the conditional check to include <code>'$(TargetArchitecture)' == 'arm' or '$(AltJitArch)' == 'arm'</code>.

Reply:
That sounds good. I'm going to add that subset to the Crossgen2 section of disabled tests.